### DISCUSSION TITLE
Update latest release to 0.2.1

### DISCUSSION THREAD

**Discussion by sokato-mdsol** [grafana/grizzly#279] [General]
The latest release is 0.2.0, whilst the document is already on 0.2.1. There are some new features released and installing the latest release (0.2.0) causes a deprecation compared to the document, such as the `Authentication` page.
https://grafana.github.io/grizzly/installation/
https://github.com/grafana/grizzly/releases

**Comment by malcolmholmes** [comment]
Yes, we're aware that the CI deploys docs on merge not on release (bad). We plan to do a release in early January that should make the code and docs line up.
